mteb/tasks/Classification/multilingual/HinDialectClassification.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks import AbsTaskClassification
+
+_LANGUAGES = {
+    "pan": ["pan-Guru"],
+    "bgc": ["bgc-Deva"],
+    "mag": ["mag-Deva"],
+    "bns": ["bns-Deva"],
+    "kfq": ["kfg-Deva"],
+    "noe": ["noe-Deva"],
+    "bhb": ["bhb-Deva"],
+    "bho": ["bho-Deva"],
+    "gbm": ["gbm-Deva"],
+    "mup": ["mup-Deva"],
+    "anp": ["anp-Deva"],
+    "hne": ["hne-Deva"],
+    "bra": ["bra-Deva"],
+    "raj": ["raj-Deva"],
+    "awa": ["awa-Deva"],
+    "guj": ["guj-Gujr"],
+    "ben": ["ben-Beng"],
+    "bhd": ["bhd-Deva"],
+    "kfy": ["kfy-Deva"],
+    "mar": ["mar-Deva"],
+    "bjj": ["bjj-Deva"],
+}
+
+
+class HinDialectClassification(AbsTaskClassification):
```

Comment:
It should inherit from `MultilingualTask` as well, so that we get results for each language.

Reply:
@isaac-chung  This is not langwise dataset. Inheriting 'MultilingualTask' throwed error. It is dialect classification. I am following SouthAfricanLangClassification, NordicLangClassification

Reply:
Oh okay! Please ignore then